extensions/warp-blink-wrtc/src/host_media/audio/loudness.rs

```diff
@@ -31,6 +31,7 @@ impl Calculator {
         f32::sqrt(self.ss * self.normalizer)
     }
 
+    #[allow(dead_code)]
```

Comment:
Just a little comment marker to come back to this in the future to determine if this will or will not be used. 

```suggestion
    //TODO: Evaluate code to determine if its needed
    #[allow(dead_code)]
```